lib/caasp.pm

```diff
@@ -33,6 +33,11 @@ sub process_reboot {
     script_run("reboot", 0) if $trigger;
 
     reset_consoles;
+    if (check_var('VIRSH_VMM_FAMILY', 'xen')) {
+        sleep 7;
+        select_console 'sut';
+    }
```

Comment:
@mnowaksuse Will I break something if I move it here? https://github.com/kravciak/openqa-tests/blob/casp-xen1/lib/opensusebasetest.pm#L218

What about timeout? - sometimes it takes longer to reboot, might be a reason to keep per-module solution. I don't feel great about moving it to wait_boot - what do you think?